build.gradle

```diff
@@ -1,23 +1,29 @@
+
 buildscript {
 	ext {
 		springBootVersion = '2.0.1.RELEASE'
 	}
-
 	repositories {
         mavenCentral()
 		jcenter()
+		maven {
+			url 'https://oss.sonatype.org/content/repositories/snapshots/'
```

Comment:
Is there a reason we are pulling in SNAPSHOT versions instead of the release version from mvnCentral or Jcenter?